spring-cloud-skipper/src/test/java/org/springframework/cloud/skipper/io/PackageWriterTests.java

```diff
@@ -88,7 +83,7 @@ private void assertExpectedContents(InputStream zipEntryInputStream, String reso
 				TestResourceUtils.qualifiedResource(getClass(), resourceSuffix)
 						.getInputStream(),
 				Charset.defaultCharset());
-		assertThat(zipEntryAsString.replace("\r\n", "\n")).isEqualTo(expectedYaml.replace("\r\n", "\n"));
+		assertEqualRemoveCr(zipEntryAsString, expectedYaml);
```

Comment:
In this case, its only used once by this test so putting inline would be fine. I will update during merge.